lms/envs/aws.py

```diff
@@ -445,6 +454,18 @@
 for database_name in DATABASES:
     DATABASES[database_name]['ATOMIC_REQUESTS'] = True
 
+# Override database settings with environment variables, if any
+for name, database in DATABASES.items():
+    if name != 'read_replica':
+        database.update({
+            'ENGINE': os.environ.get('DB_ENGINE', database['ENGINE']),
```

Comment:
@feanil understood. I have switched it back to use the `DB_MIGRATION_*` environment variables:

- https://github.com/edx/edx-platform/commit/1232792afa690520a90802ea5577cfbd46a422d5
- https://github.com/edx/configuration/commit/7971ff905cbf7b2ad423694891755eb3efbfa291
- https://github.com/open-craft/xqueue/commit/2078e54bb275f97fee75b1f1b238aa5d5a444a7b